.github/workflows/test-reporter.yml

```
@@ -16,5 +16,26 @@ jobs:
         with:
           artifact: test-results            # artifact name
           name: JUnit Tests                 # Name of the check run which will be created
-          path: "**/*.xml"                  # Path to test results (inside artifact .zip)
+          path: "**/elastic-serverless-forwarder-junit.xml"                  # Path to test results (inside artifact .zip)
           reporter: java-junit              # Format of test results
+          output-to: step-summary           # Write summary in the PR
+  coverage:
+    if: ${{ github.event.workflow_run.event == 'pull_request' }}
+    steps:
```

Jobs need the `runs-on`:

```suggestion
    runs-on: ubuntu-latest
    steps:
```